app/views/events/show.html.erb

```diff
@@ -9,3 +9,6 @@
   <dt>Location</dt>
   <dd><%= @event.location %></dd>
 </dl>
+
+<%= link_to 'Edit', edit_event_path(@event) %>
+<%= render partial: 'delete-link', locals: { event: @event } %>
```

Comment:
The delete link partial was created in the previous commit (Add Delete links to #index).